apps/mochi-web/components/app/StatisticsBox.tsx

```diff
@@ -0,0 +1,64 @@
+import { Typography } from '@consolelabs/core'
+import { ArrowUpLine } from '@consolelabs/icons'
+import clsx from 'clsx'
+import { formatNumber } from '~utils/number'
+
+interface Props {
+  label: string
+  amount?: number
+  formatAmount?: (amount: number) => string
+  change?: number
+  formatChange?: (change: number) => string
+  milestone?: 'week' | 'month'
+  size?: 'md' | 'lg'
+  footer?: React.ReactNode
+  className?: string
+}
+
+export const StatisticsBox = ({
+  label,
+  amount = 0,
+  formatAmount = formatNumber,
+  change = 0,
+  formatChange = (change) => `${Math.abs(change) * 100}%`,
+  milestone = 'month',
+  size = 'md',
+  footer,
+  className,
+}: Props) => {
+  return (
+    <div
+      className={clsx(
+        'bg-neutral-0 rounded-xl',
+        size === 'md' ? 'p-4 space-y-4' : 'p-6 space-y-2',
+        className,
+      )}
+    >
+      <Typography level="h8" color="textPrimary">
```

Comment:
We recently experienced a bug with the font weight, which required me to fix it through two PRs. Can you please double-check to ensure this issue does not occur again?

- https://github.com/consolelabs/web-foundation/pull/313
- https://github.com/consolelabs/web-foundation/pull/312